### PR TITLE
Jionghao: Add Experience section

### PR DESCRIPTION
**Pull request by Jhwu3**
L=ENG-109

### CODE REVIEW

**Comment by Jhwu3** [comment]
image

File: js/src/personal/jionghao/components/Experience.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { Title, Text, Tabs, List } from '@mantine/core'
```
Comment (arklian) [inline js/src/personal/jionghao/components/Experience.tsx:1]: Break this up into multiple components perhaps? It's getting a bit longer.

File: js/src/personal/jionghao/components/Experience.module.css [hunk 2]
```diff
@@ -6,11 +6,29 @@
 }
 
 .verticalTab{
-  padding-right: 4vw;
-  padding-top: 2vh;
-  padding-bottom: 2vh;
+  padding-right: 3vw;
```
Comment (arklian) [inline js/src/personal/jionghao/components/Experience.module.css:9]: Stick with bigger numbers if you're using vh/vw. 
If it's this small, use rem/px

File: js/src/personal/jionghao/components/Experience.module.css [hunk 3]
```diff
@@ -6,11 +6,29 @@
...
+  padding-bottom: 1.5vh;
 }
 
 .tabPanel{
   width: 40vw;
+  min-height: 200px;
+}
+
+.tabsList{
+  height:100%;
```
Comment (arklian) [inline js/src/personal/jionghao/components/Experience.module.css:20]: nit: space

File: js/src/personal/jionghao/components/Experience.tsx [hunk 4]
```diff
@@ -7,86 +8,180 @@ export function Experience() {
...
         color="cyan"
         orientation="vertical"
-        defaultValue="gallery"
+        defaultValue="Patina"
         placement="right"
+        radius={'xs'}
       >
         <Tabs.Panel
-          pl={'2vh'}
+          pl={'2vw'}
```
Comment (arklian) [inline js/src/personal/jionghao/components/Experience.tsx:23]: 2vh

File: js/src/personal/jionghao/components/Work.tsx
Comment (arklian) [inline js/src/personal/jionghao/components/Work.tsx:1]: I don't think you should be adding empty files.

File: js/src/personal/jionghao/components/Work.module.css [hunk 5]
```diff
@@ -0,0 +1,30 @@
+.work{
+  display: flex;
+  align-self: center;
+  flex-direction: column;
+  margin-bottom: 30vh;
```
Comment (arklian) [inline js/src/personal/jionghao/components/Work.module.css:5]: I feel like you overuse vh for a lot of things

**Review by arklian — APPROVED** [review]

**Pull request merged**